extensions/amp-story/1.0/animation.js

```diff
@@ -59,7 +64,10 @@ const DEFAULT_EASING = 'cubic-bezier(0.4, 0.0, 0.2, 1)';
  * TODO(alanorozco): maybe memoize?
  */
 export function hasAnimations(element) {
-  return !!scopedQuerySelector(element, ANIMATABLE_ELEMENTS_SELECTOR);
+  return (
+    !!childElementByTag(element, 'amp-story-animation') ||
+    !!scopedQuerySelector(element, ANIMATABLE_ELEMENTS_SELECTOR)
+  );
```

Comment:
Nit, if like me you think it's easier to read:

```suggestion
  return !!scopedQuerySelector(element, `${ANIMATABLE_ELEMENTS_SELECTOR}, amp-story-animation`);
```